page/blocks/person.ejs

```diff
@@ -9,5 +9,10 @@
     <p class="bio hidden">
         <%= bio %>
     </p>
-    <span class="pre-title hidden"><%if (typeof preTitle != 'undefined') {%>  <%= preTitle %>  <%}%></span>
```

Comment:
This is from the design.
<img width="872" alt="screen shot 2017-08-23 at 3 57 51 pm" src="https://user-images.githubusercontent.com/1808867/29619642-de6e376c-881b-11e7-9a42-464373508b9f.png">

check the word "Partner"